docs/conceptual-design.md

```diff
@@ -4,74 +4,95 @@ This is a working document intended to capture key concepts and terminology that
 
 As these concepts are used throughout Cherrytwist, it is important for anyone new to Cherrytwist to be familiar with the contents of this document.
 
-# **Challenge: A Journey with many steps**
+# **Challenge: A Journey towards a desired outcome**
 
-The central concept in Cherrytwist is the *Challenge* - a complex problem / change to be realised and involving multiple stakeholders. 
+The central concept in Cherrytwist is the *Challenge* - a desired outcome to be realised that requires multiple parties to collaborate over a longer period of time.  
 
-It is a _*journey*_, whereby those invovled in the challenge will need to work together to realise steps in the right direction. 
+It is a _*journey*_, whereby those invovled in the challenge will need to work together to realise steps in the right direction towards a desired outcome. 
 
-![Challenge as a journey with multiple steps](Images/ChallengeJourney.png "Challenge as a journey with multiple steps")
+<p >
+<img src="visuals/challenge-journey.jpg" alt="Challenge as a journey towards a desired outcome" width="600" />
+</p>
 
-All other concepts / activities in Cherrytwist are to support the realisation of Challenges - from the activities on the journey, to the community that builds up around the Challenge. In some sense it is the placement of the Challenge as being _the_ heart of Cherrytwist that sets it apart.
+All other concepts / activities in Cherrytwist are to support the realisation of Challenges - from the activities on the journey, to the community that builds up around the Challenge, to the collaboration that takes place around the shared representation of the Challenge. 
 
-## Projects: Steps on the Journey
-The term used within Cherrytwist for the steps on the Challenge journey is *Project*, whereby a set of participants collaborate, within the context of a Challenge, towards a clearly defined outcome. 
+And with any journey the _*Map*_ you have of where you are heading is critical to getting you towards your destination as well as finding the right companions for the journey. This needs to be a shared representation - as opposed to the current default reality whereby all parties have pieces of the puzzle that is the map. 
```

Comment:
This paragraph is a bit confusing. Maybe introduce the concept of the map first and then talk about it? Also try using sorter sentences.